src/renderer/coremods/commands/commands.ts

```diff
@@ -0,0 +1,382 @@
+
+          choices.push(
+            ...enabledPlugins.map((plugin) => ({
+              name: plugin.manifest.name,
+              displayName: `Plugin: ${plugin.manifest.name}`,
+              value: plugin.manifest.id,
+            })),
+          );
+          choices.push(
+            ...enabledThemes.map((theme) => ({
+              name: theme.manifest.name,
+              displayName: `Theme: ${theme.manifest.name}`,
+              value: theme.manifest.id,
+            })),
+          );
+
+          return choices;
+        },
+      },
+    ],
+    executor: async (args) => {
+      try {
+        if (plugins.plugins.has(args[0].value as string)) {
+          await plugins.reload(args[0].value as string);
+        } else {
+          themes.reload(args[0].value as string);
+        }
+        return {
+          send: false,
+          embeds: [
+            {
+              type: "rich",
+              color: 0x1bbb1b,
+              title: "Success",
+              description: `${args[0].value} has been reloaded!`,
+            },
+          ],
+        };
+      } catch (err) {
+        return {
+          send: false,
+          embeds: [
+            {
+              type: "rich",
+              color: 0xdd2d2d,
+              title: "Error",
+              description: err as string,
+            },
+          ],
+        };
+      }
+    },
+  },
+  {
+    name: "list",
+    description: "List all plugins/themes",
+    usage: '/list <true/false> <"plugin"/"theme"> <>',
+    options: [
+      {
+        name: "send",
+        description: "Whether you want to send this or not.",
+        type: ApplicationCommandOptionType.Boolean,
+        required: false,
+      },
+      {
+        name: "addon type",
+        description: "Choose what type of addons to list.",
+        type: ApplicationCommandOptionType.String,
+        required: true,
+        choices: [
+          {
+            name: "Theme",
+            displayName: `List themes`,
+            value: "theme",
+          },
+          {
+            name: "Plugin",
+            displayName: `List plugins`,
+            value: "plugin",
+          },
+        ],
+      },
+      {
+        name: "version",
+        description: "Whether you want to add version info.",
+        type: ApplicationCommandOptionType.Boolean,
+        required: true,
+      },
+      {
+        name: "type",
+        description: "Whether you want to send either only enabled, disabled or all themes.",
+        type: ApplicationCommandOptionType.String,
+        required: true,
+        choices: [
+          {
+            name: "Enabled",
+            displayName: "Enabled",
+            value: "enabled",
+          },
+          {
+            name: "Disabled",
+            displayName: "Disabled",
+            value: "disabled",
+          },
+          {
+            name: "Both",
+            displayName: "Both",
+            value: "default",
+          },
+        ],
```

Comment:
This option is unnecessary, just always include both.

Reply:
not really. People sometimes don't wanna share disabled plugins coz question morals shit there so better keep this and default to both making it not required

Reply:
No

Reply:
whats the issue with having an optional option?

Reply:
The option is required...

Reply:
Fine. But the name needs to change.

Reply:
changed the name to status